spec/index.bs

```diff
@@ -13600,16 +13729,14 @@ satisfy all validation requirements. Validation errors are always indicative of
 error, and is expected to fail the same way across all devices assuming the same
 {{device/[[features]]}} and {{device/[[limits]]}} are in use.
 
-<div algorithm>
+<div algorithm data-timeline=content>
```

Comment:
I think this should be a device-timeline algorithm

Reply:
Huh. Yeah, it should be. Already had "content timeline" in the steps, so I didn't question it. Thanks!

Reply:
All the more reason to validate timeline usages in the javascript :)